# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xpcom/xpcAccessibleTableCell.cpp
# Commit: 48dc94da9565
# Full Hash: 48dc94da9565d7f804092209a27821d729ccde60
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-04-02 09:44:13
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r 477c59860bba -r 48dc94da9565 accessible/xpcom/xpcAccessibleTableCell.cpp
--- a/accessible/xpcom/xpcAccessibleTableCell.cpp	Fri Apr 01 09:49:56 2022 +0000
+++ b/accessible/xpcom/xpcAccessibleTableCell.cpp	Fri Apr 01 09:49:57 2022 +0000
@@ -94,7 +94,7 @@
 
   if (!Intl()) return NS_ERROR_FAILURE;
 
-  AutoTArray<LocalAccessible*, 10> headerCells;
+  AutoTArray<Accessible*, 10> headerCells;
   Intl()->ColHeaderCells(&headerCells);
 
   nsCOMPtr<nsIMutableArray> cells = do_CreateInstance(NS_ARRAY_CONTRACTID);
@@ -115,7 +115,7 @@
 
   if (!Intl()) return NS_ERROR_FAILURE;
 
-  AutoTArray<LocalAccessible*, 10> headerCells;
+  AutoTArray<Accessible*, 10> headerCells;
   Intl()->RowHeaderCells(&headerCells);
 
   nsCOMPtr<nsIMutableArray> cells = do_CreateInstance(NS_ARRAY_CONTRACTID);